yabt/dot.py

```diff
@@ -59,6 +60,34 @@ def get_not_buildenv_targets(build_context):
     return visited
 
 
+def dot_file_line_proc(nx_g, line):
+    """ Parsing dot file line """
```

Comment:
Please add to the doc an example of how does this line look, it will make it much easier to read this code